### PR TITLE
Yuliu2/add purview configuration

### PR DESCRIPTION
**Pull request by liuyu1981831m**
Add PurviewConfiguration property in ADF

### CODE REVIEW

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json [hunk 1]
```diff
@@ -4983,6 +4983,16 @@
         "type": "object"
       }
     },
+    "FactoryPurviewConfiguration": {
```
Comment (Frey-Wang) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json:4986]: Please keep this property name the same as in Synapse as well: "PurviewConfiguration".

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json [hunk 2]
```diff
@@ -5190,6 +5200,10 @@
           "type": "string",
           "readOnly": true
         },
+        "purviewConfiguration": {
+          "$ref": "#/definitions/FactoryPurviewConfiguration",
```
Comment (Frey-Wang) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json:5204]: Keep the same as in Synapse

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json (hunk 1 above)
Comment (liuyu1981831m) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json]: Fixed

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json (hunk 2 above)
Comment (liuyu1981831m) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json]: Fixed

**Review by Frey-Wang — APPROVED** [review]

**Comment by liuyu1981831m** [comment]
@leni-msft Could you please help merge the PR?

**Pull request merged**